jenkins/catapult/process.py

```diff
@@ -9,6 +9,13 @@ class ProcessManager:
 	def __init__(self, dry_run=False):
 		self.dry_run = dry_run
 
+	@staticmethod
+	def _decode_line(line_bin):
+		try:
+			return line_bin.decode('utf-8')
+		except UnicodeDecodeError as ex:
+			return f'Failed to decode line: {line_bin}\n Exception: {str(ex)}\n'
```

Comment:
is it possible to just strip out / replace binary characters?

Reply:
imo ignoring the exception is kinda fine, as it should happen only on windows... (unless we spit out some crap, somewhere), however...

@Wayonb hm, the actual reason for that error is that this obviously is _not_ UTF8 sequence, 0xFB is invalid prefix, I'm wondering if this is something GTEST spits out (as this line is likely colored, although I don't see how this should be related...) or is it something else?
You might want to try adding `--gtest_color=no` on windows though, maybe it will help?

The other possible issue (although not sure if this will help or make things worse): is console codepage set to utf8? (there's `chcp` thing: https://superuser.com/a/269857)

Reply:
@gimre-xymcity You are right in that its something to do with GTEST but it seems random, meaning it wasnt always the same test that cause this error.

I will try turning off the color for gtest on Windows.  Thanks.

Reply:
I think disabling the color on gtest might have fix the issue.  Its hard to tell since it does happened all the time.

I also left logic in there to just drop characters which it fails to decode.  There is a perf hit with this so maybe just do it for Windows?

Reply:
Still repo with gtest color disabled